index.html

```diff
@@ -223,5 +223,49 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+
+    <footer id="contact-form" class="flex-container">
+      <h1>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h1>
+
+      <form action="https://formspree.io/f/xeqdoera" method="POST">
+        <input type="text" name="fullname" id="fullname" placeholder="Full name" maxlength="30" required>
+        <input type="email" name="email" id="email" placeholder="Email address" required>
+        <textarea name="message" id="massage" maxlength="500" placeholder="Enter text here" required ></textarea>
+        <button class="call-to-action" type="submit">Get in touch</button>
```

Comment:
Great design @laiifuu 🤝

- [ ] As stated in the project requirements, kindly implement all the button interactions of `enable`, and  `hover`.